Dockerfile.in

```diff
@@ -23,7 +23,13 @@ RUN apk update --no-cache && apk add \
     ca-certificates \
     coreutils \
     git \
-    openssh-client
+    openssh-client \
+    wget
+
+RUN wget -O /sbin/tini https://github.com/krallin/tini/releases/download/v0.14.0/tini-static-ARG_ARCH && \
```

Comment:
I don't think we should be pulling un-managed binary files like this.  No verification of what we're getting at all.

My preference would be to solve this inside git-sync itself - it's not that complex.